Cargo.toml

```diff
@@ -77,6 +77,12 @@ passthrough-decoder = ["librespot-playback/passthrough-decoder"]
 
 default = ["rodio-backend"]
 
+[profile.release-dist-optimized]
+inherits = "release"
+panic = "abort"
```

Comment:
I'd recommend keeping `unwind` to drop and free what we can. Should be cleaner.

Reply:
Abort as far as I know does that it just doesn't unwind and it exits as you's expect a program to do when it crashes. Having librespot just hang when it crashes puts a really big kink in using it as a service (which is the way most people use it).

Reply:
There is also this: https://doc.rust-lang.org/nomicon/unwinding.html

Which seems to indicate that aborting is the right thing to do when a program catastrophically fails not unwinding.

Reply:
OK sounds good.